lib/ext/measure.rb

```diff
@@ -14,7 +14,10 @@ def display_name
   end
 
   def cms_int
```

Comment:
What is this used for?  Also, if there is more than 1 measure without a cms id then they all will be the same cms_int "0"